bats/tests/k8s/wasm.bats

```diff
@@ -0,0 +1,132 @@
+load '../helpers/load'
+
+local_setup() {
+    if using_docker; then
+        skip "this test only works on containerd right now"
+    fi
+}
+
+assert_traefik_crd_established() {
+    local jsonpath="{.status.conditions[?(@.type=='Established')].status}"
+    run kubectl get crd traefikservices.traefik.containo.us -o jsonpath="$jsonpath"
+    assert_success || return
+    assert_output 'True'
+}
+
+@test 'start k8s without wasm support' {
+    factory_reset
+    start_kubernetes
+    wait_for_kubelet
+
+    # The manifests in /var/lib/rancher/k3s/server/manifests are processed
+    # in alphabetical order. So when traefik.yaml has been loaded we know that
+    # rd-runtime.yaml has already been processed.
+    try assert_traefik_crd_established
+}
+
+@test 'verify no runtimeclasses have been defined' {
+    run kubectl get runtimeclasses -o json
+    assert_success
+
+    run jq_output '.items | length'
+    assert_success
+    assert_output 0
+}
+
+@test 'start k8s with wasm support' {
+    # TODO We should enable the wasm feature on a running app to make sure the
+    # TODO runtime class is defined even after k3s is initially installed.
+    factory_reset
+    start_kubernetes --experimental.container-engine.web-assembly.enabled
+    wait_for_kubelet
+    try assert_traefik_crd_established
+}
+
+@test 'verify spin runtime class has been defined (and no others)' {
+    run kubectl get runtimeclasses -o json
+    assert_success
+
+    rtc=$output
```

Comment:
We don't use `local` in `@test` blocks because they are running in a separate subshell each anyways. Just like you don't use `local` at the file scope (well, and can't, in that case).